imports/ui/views/login/LoginModal/index.js

```diff
@@ -4,24 +4,49 @@ import Modal from 'react-modal'
 import EmailPasswordLogin from '/imports/ui/views/login/LoginModal/EmailPasswordLogin.jsx'
 import SSOButtons from '/imports/ui/views/login/SsoButtons.jsx'
 import CreateAnAccountLogin from '/imports/ui/views/login/LoginModal/CreateAnAccountLogin.jsx'
+import { Meteor } from 'meteor/meteor'
+import { browserHistory } from 'react-router'
 
 export default class LoginModal extends Component {
+  constructor(props) {
+    super(props)
+    let isUserLoggedIn = Meteor.user() != null
```

Comment:
I'd move this line into a `createContainer` outside of the modal, then pass in the user as a prop.